kie-server-parent/kie-server-remote/kie-server-graphql/kie-server-graphql-jbpm/src/main/resources/jbpm-schema.graphql

```diff
@@ -0,0 +1,270 @@
+scalar Date
+scalar Object
+
+schema {
+  # Root of Queries
+  query: Query
+  # Root of Mutations
+  mutation: Mutation
+}
+
+type Query {
+  """
+  Root for process definition related queries
+  """
+  definitions: DefinitionQuery
+  """
+  Root for process instance related queries
+  """
+  instances: InstanceQuery
+}
+
+type Mutation {
+
+  """
+  Root for process instance related mutations
+  """
+  instances: InstanceMutation
+}
+
+"""
+Collection of every query supported for ProcessDefinitions
+"""
+type DefinitionQuery {
+  """
+  Gets ProcessDefinition with specified processDefinitionId and containerId.
+  """
+  processDefinition(processDefinitionId: String!, containerId: String!): ProcessDefinition!
+
+  """
+  Gets list of UserTaskDefinition's for with specified processDefinitionId and containerId.
+  """
+  userTaskDefinitions(processDefinitionId: String!, containerId: String!): [UserTaskDefinition]
+
+  """
+  Gets all available process definitions from runtime.
+  """
+  allProcessDefinitions(batchSize: Int): [ProcessDefinition]
+
+  """
+  Gets a list of ProcessDefinition based no arguments.
+  """
+  getProcessDefinitions(processDefinitionId: String, containerId: String): [ProcessDefinition]
```

Comment:
so the difference between this and the processDefinition is that it can either take process definition by id or by container but not both right?